front/src/views/routes/common/cms/CommonCMSPage.vue

```diff
@@ -27,18 +27,19 @@
 
       <div class="lck-container-parent p-mx-2">
         <div
-          v-for="container in displayContainersByModeNavigation"
+          v-for="container in this.page.containers"
```

Comment:
Why did you remove the `displayContainersByModeNavigation` ?

Reply:
Because now, we have a v-show